boss-ws/src/main/java/org/genomebridge/boss/http/resources/ObjectResource.java

```diff
@@ -167,10 +167,24 @@ public String delete(@ApiParam(required = true,value = "Object Boss Id")
         return objectId;
     }
 
-    @Path("multi")
+    @Path("/multi")
     @Produces("application/json")
     @POST
-    public CopyResponse getUploadID(@PathParam("objectId") String objectId,
+    @ApiOperation(value = "Resolving Objects - Resumable Uploads extension",
+                  notes = "Objects are 'resolved' into an URL which can be use to upload a resource of indefinitely large size. For an object, multi will return a GCS Resumable Upload URL.",
```

Comment:
please change the notes attribute of the annotation to something like:
"Objects are 'resolved' into a GCS Resumable Upload URL which can be use to upload a resource of indefinitely large size in multiple parts. Applies only to objects using the cloudStore storage platform."